sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -846,3 +846,19 @@ def merge(self, url, params=None, headers=None, content=None, form_content=None)
             "MERGE", url, params, headers, content, form_content, None
         )
         return request
+
+    def options(self, url, params=None, headers=None, content=None, form_content=None):
+        # type: (str, Optional[Dict[str, str]], Optional[Dict[str, str]], Any, Optional[Dict[str, Any]]) -> HttpRequest
+        """Create a OPTIONS request object.
+
+        :param str url: The request URL.
+        :param dict params: Request URL parameters.
+        :param dict headers: Headers
+        :param dict form_content: Form content
```

Comment:
I would make the form_content a `kwargs`. To the best of my knowledge, there is no defined semantics of what it means to send a body in a options request.

Reply:
So I guess your comment then goes for content and form_content?
This would mean that I need to do a special case in the code generator, or decide to pass content always as kwargs to avoid a "if OPTIONS: call this way; else call this way". I'm not sure it worth it?

Reply:
Are we always using positional parameters in the generated code? If so, shouldn't we use named parameters on the call site?

Reply:
In this case, for some reason `content` is positional and `form_content` is kwarg. I didn't challenge how autorest was generating this signature, I can change all content to kwargs, I'm fine with that.

Reply:
pushed content / form_content as kwargs